src/main/java/io/neonbee/data/internal/metrics/DataVerticleMetrics.java

```diff
@@ -0,0 +1,50 @@
+package io.neonbee.data.internal.metrics;
+
+import java.util.List;
+
+import io.micrometer.core.instrument.Tag;
+import io.vertx.core.Future;
+
+public interface DataVerticleMetrics {
+
+    /**
+     * Reports the sum of all calls.
+     *
+     * @param name        the name of the metric
```

Comment:
```suggestion
     * @param name        name of the metric
```

for all, to make param description consistent